myapp/templates/financial.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{#This data needs to be changed as needed#}
+{% block content %}
+<title>Financial Report</title>
+    <div class="col-md-12 text-left">
+        <h2>Financial Report</h2>
+		<h3 class="display-4">
+				{{ data.symbol }}
+				<small class="text-muted">({{ data.symbol }})</small>
```

Comment:
can you reformat this a bit? there is auto formatting in pycharm (and almost any other ide )

Reply:
This is totally deleted.